.github/workflows/_rocm-test.yml

```diff
@@ -170,9 +170,7 @@ jobs:
           SHARD_NUMBER: ${{ matrix.shard }}
           NUM_TEST_SHARDS: ${{ matrix.num_shards }}
           REENABLED_ISSUES: ${{ steps.keep-going.outputs.reenabled-issues }}
-          SCCACHE_BUCKET: ossci-compiler-cache-circleci-v2
```

Comment:
It's easier to just not use S3 cache bucket on ROCm runner.  The runner will just use the cache locally instead.  The assumption here is that test job wouldn't need to build many things anyway (mainly some test extension)

If the duration becomes worse, I could update the OIDC role to grant it write access to S3 instead.